aries_cloudagent/anoncreds/issuer.py

```diff
@@ -632,3 +632,62 @@ async def create_credential(
             raise AnonCredsIssuerError("Error creating credential") from err
 
         return credential.to_json()
+
+    async def create_credential_vc_di(
```

Comment:
is this called?  @ianco i am not sure if there is name magic, if not i don't see where this is called - perhaps if we call it from the appropriate place - 
@sarthakvijayvergiya i hadn't thought about that, can we maybe see if we can just call this function to create the credential??

Reply:
have deleted this, it was WIP